### PR TITLE
4r - show list of all tables

### PR DESCRIPTION
**Pull request by MaksKulykov**

### CODE REVIEW

File: src/maks.kulykov/DatabaseManager.java [hunk 1]
```diff
@@ -37,4 +36,31 @@ public String connection() {
...
         return responseMessage;
     }
+
+    public String getTablesList() {
+        StringBuilder tableList = new StringBuilder();
+        ResultSet rs = null;
+        try {
+            DatabaseMetaData dbmd = conn.getMetaData();
+            rs = dbmd.getTables(null, null, "%", new String[] { "TABLE" });
+            tableList = new StringBuilder("[");
```
Comment (OlgaKulikova) [inline src/maks.kulykov/DatabaseManager.java]: don't add new stringbuilder here

File: src/maks.kulykov/DatabaseManager.java [hunk 2]
```diff
@@ -37,4 +36,31 @@ public String connection() {
...
+            while (rs.next()) {
+                tableList.append(rs.getString("TABLE_NAME"));
+                tableList.append(", ");
+            }
+            tableList.delete(tableList.toString().length() - 2, tableList.toString().length());
```
Comment (OlgaKulikova) [inline src/maks.kulykov/DatabaseManager.java]: move `tableList.toString().length()` to the separate variable and check the length before remove length-2.

File: src/maks.kulykov/DatabaseManager.java (hunk 1 above)
Comment (MaksKulykov) [reply src/maks.kulykov/DatabaseManager.java]: +

File: src/maks.kulykov/DatabaseManager.java (hunk 2 above)
Comment (MaksKulykov) [reply src/maks.kulykov/DatabaseManager.java]: +

**Pull request merged**